### DISCUSSION TITLE
Dependent Select Options not working

### DISCUSSION THREAD

**Discussion by almasrient** [filamentphp/filament#10335] [Help] [answered]
Hi,

I am trying to follow the instructions from this link
https://filamentphp.com/docs/3.x/forms/advanced#dependant-select-options

but I am having a problem with this error message.

![image](https://github.com/filamentphp/filament/assets/140850598/0ef44989-0ccb-4ace-bb59-40b228fb30d9)

Here is my code
https://github.com/almasrient/MMS-F

Explanation is appreciated.

**Comment by ShaungBhone** [comment]
You need to import your Negeri Resources.

**Reply by almasrient** [reply]
My Negeri Resources named as below
use App\Filament\Resources\NegeriResource;

So, it does not match with the Class name.
But, when I change the Class name to the NegeriResources, 
here is the error message

![image](https://github.com/filamentphp/filament/assets/140850598/9ada069f-d075-4c6c-8b29-52751ec40221)

**Reply by ShaungBhone** [reply]
I did not see your code.

**Reply by almasrient** [reply]
Let me check the access.

**Reply by almasrient** [reply]
Done, change from Private to Public. Can you confirm?

**Reply by ShaungBhone** [reply, marked as answer]
```
use App\Models\Negeri;
```
need to import your class

**Reply by almasrient** [reply]
Okay, it is worked now.

Only required to import
use App\Models\Negeri;

without changing the class name (from Negeri to NegeriResource).

My mistakes was, I thought when there are relationships already called, and the BelongsTo class been developed, it is automatically identify the Model class.

Thank you ShaungBhone.

**Reply by ShaungBhone** [reply]
Good to help
